AdobeStockAsset/etc/db_schema.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="adobe_stock_request" resource="default" engine="innodb" comment="Adobe Stock License Request">
+        <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true" comment="Entity ID"/>
+        <column xsi:type="int" name="admin_user_id" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Admin User Id"/>
+        <column xsi:type="varchar" length="255" name="request_id" nullable="false" comment="Request ID"/>
+        <constraint xsi:type="primary" referenceId="PRIMARY">
+            <column name="id"/>
+        </constraint>
+        <constraint xsi:type="foreign" referenceId="ADO_STO_LIC_REQ_ADM_USER_ID" table="adobe_stock_request" column="admin_user_id" referenceTable="admin_user" referenceColumn="user_id" onDelete="CASCADE"/>
+    </table>
+    <table name="adobe_stock_user_profile" resource="default" engine="innodb" comment="Adobe Stock User Profile">
+        <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true" comment="Entity ID"/>
+        <column xsi:type="int" name="adobe_id" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Adobe Id"/>
+        <column xsi:type="int" name="admin_user_id" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Admin User Id"/>
+        <column xsi:type="varchar" length="255" name="display_name" nullable="false" comment="Display Name"/>
+        <column xsi:type="varchar" length="255" name="preferred_languages" nullable="false" comment="Preferred Languages"/>
```

Comment:
I think we should make preferred_languages a foreign key and introduce an appropriate table

Reply:
@sivaschenko What would be the best definition for this `preferred languages` table?
table name: adobe_stock_preferred_languages
column name: id, pk: yes, type: int
column name: language, type: varchar, length: 255
???

Reply:
Yes, sound good to me! I'd shorten the table name to `adobe_stock_language` though

Reply:
@sivaschenko I have added a separated table to link languages and users as requested.